macosx/Controller.mm

```diff
@@ -5179,27 +5179,27 @@ - (void)setWindowSizeToFit
 
         if (![self.fDefaults boolForKey:@"AutoSize"])
         {
-            //only set a minimum height constraint
+            // Only set a minimum height constraint
             CGFloat height = self.minScrollViewHeightAllowed;
             NSString* constraintsString = [NSString stringWithFormat:@"V:[scrollView(>=%f)]", height];
             self.fStackViewHeightConstraints = [NSLayoutConstraint constraintsWithVisualFormat:constraintsString options:0
-                                                                                       metrics:nil
-                                                                                         views:views];
+                                                                                    metrics:nil
+                                                                                        views:views];
```

Comment:
kindly rollback those two lines: the change in spaces is breaking the clang-format test